### DISCUSSION TITLE
v9 status / v9 docs

### DISCUSSION THREAD

**Discussion by csenio** [pmndrs/react-spring#1104] [Support] [answered]
Is v9 still in active development? Are the new APIs / changes documented somewhere and is it stable enough to use in production?

**Comment by aleclarson** [comment, marked as answer]
> Is v9 still in active development?

Yes, although I've recently been too busy to consistently release new RC versions.

> Are the new APIs / changes documented somewhere

v9 summary: https://aleclarson.github.io/react-spring/v9/
v9 feedback: #985



> is it stable enough to use in production?

I'd say so. I'm using it without issue in my projects.
